RELEASE/scripts/autoscend.ash

```diff
@@ -1336,10 +1336,23 @@ void beatenUpResolution()
 
 	if(have_effect($effect[Beaten Up]) > 0)
 	{
-		cli_execute("refresh all");
-		if(have_effect($effect[Beaten Up]) > 0)
+		if(have_effect($effect[Beaten Up]) == 2 && get_property("lastEncounter") == "Dr. Awkward" && internalQuestStatus("questL11Palindome") > 5)
```

Comment:
I haven't looked super close at this quest in a while so bear with me. 

Isn't quest status > 5 mean we already defeated dr awkward? Step 1 is when photos have been arranged and that is when we get beaten up by the quest item.

Reply:
> Isn't quest status > 5 mean we already defeated dr awkward

yes it is this situation:
photos have been arranged and we get beaten up by the quest item
autoscend does not manage to remove beaten up
it still fights dr awkward and wins
[Beaten Up]) == 2
autoscend thinks it was beaten up by a monster and now wants to abort but that's not what happened so aborting is not necessary in this case

Reply:
Ah that makes sense, ty